app/src/main/java/com/pinomg/determinator/api/ApiHandler.java

```diff
@@ -23,7 +23,7 @@
  */
 public class ApiHandler {
     
-    private static final String BASE_URL = "http://95.80.41.105:8004/determinator_server/";
+    private static final String BASE_URL = "http://192.168.0.11/pinomg/";
```

Comment:
Okey, but then we have to change it back after the server is updated. And that's annoying! ;)